### ISSUE TITLE
wk2 apr_epic Bug - Fuzzy QA Story 4 Graphic in the Team PSD Manual Chapter 11.2

### ISSUE THREAD

**Issue by lzim** [lzim/teampsd#1864]
Please upload a fresh image 😸 
It's fuzzy 🔍 

https://lzim.github.io/teampsd/11-track-issues-features.html#quality-control-review-cycle
 
![image](https://user-images.githubusercontent.com/7454688/113637492-930f2180-9629-11eb-81e5-29b6a5a52550.png)

**Comment by jamesmrollins** [comment]
Hi @lijenn!  Can you remind me where to find the diagram above?  I have looked through Lucid charts with no luck.  Thank you, James

**Comment by lijenn** [comment]
Hi @jamesmrollins! Right here, it's in a tab in the Story Map: https://lucid.app/lucidchart/4fb06b59-38f5-4d61-86f0-4620ceb93196/edit?page=Ky670QAJ06jG#

**Comment by jamesmrollins** [comment]
Hi @lijenn and @lzim 
I spent an hour on this problem this morning and attempted to improve the image quality.  I tried recapturing a larger screen rendering, I saved as a print quality PNG and redrew the image in a new document and saved as a print quality PNG.  I then added the image back to the MD and rendered it in bookdown.  It seems to have improved a bit.  I am not sure what else I could do to improve the image quality. ~James
